WalletWasabi.Tests/UnitTests/Transactions/PayjoinTests.cs

```diff
@@ -464,7 +464,7 @@ public void BrokenPayjoinServerTest()
 			}
 			var coinsView = new CoinsView(scoins);
 
-			var bitcoinStore = new BitcoinStoreMock();
+			var bitcoinStore = new BitcoinStore(null!, new AllTransactionStoreMock(workFolderPath: ".", Network.Main), null!);
```

Comment:
I understand this is a test but we should have in mind that the same conventions and code quality have to apply to all code. From this point of view we would never accept this in wasabi code so, we must reject it here too.

Reply:
Just for the sake of completeness: This is what `BitcoinStoreMock` did but it was far from apparent.

What is your suggestion to make it nicer?

btw: WalletWasabi/Stores/BitcoinStoreMock.cs  - I don't think the mock belong to WalletWasabi project, so I think it is a win that the file was deleted.

Reply:
It seems the `TransactionFactory` doesn't need a `BitcoinStore` but its internal `TransactionStore` instead. I think a small refactoring could solve this problem.

Reply:
Ah. good catch.